stellarphot/photometry/photometry.py

```diff
@@ -380,7 +393,11 @@ def single_image_photometry(
     if use_coordinates == "sky":
         try:
             xcen, ycen = centroid_sources(
-                ccd_image.data, xs, ys, box_size=2 * photometry_apertures.radius + 1
+                # TODO: FIX THIS
```

Comment:
Is this a reminder that you need finish something here?

Reply:
I think it was just a reminder to change the aperture radius to be in pixels, i.e. to use `radius_pixel` instead of `radius`.

Reply:
Removed it.